ddl/placement_sql_test.go

```diff
@@ -17,21 +17,28 @@ package ddl_test
 import (
 	"fmt"
 	"sort"
+	"testing"
 
-	. "github.com/pingcap/check"
 	"github.com/pingcap/failpoint"
 	"github.com/pingcap/tidb/ddl"
 	"github.com/pingcap/tidb/ddl/placement"
+	"github.com/pingcap/tidb/domain"
 	mysql "github.com/pingcap/tidb/errno"
 	"github.com/pingcap/tidb/parser/model"
 	"github.com/pingcap/tidb/session"
-	"github.com/pingcap/tidb/util/testkit"
+	"github.com/pingcap/tidb/testkit"
 	"github.com/pingcap/tidb/util/testutil"
+	"github.com/stretchr/testify/require"
 )
 
 // TODO: Remove in https://github.com/pingcap/tidb/issues/27971 or change to use SQL PLACEMENT POLICY
-func (s *testDBSuite1) TestPlacementPolicyCache(c *C) {
-	tk := testkit.NewTestKit(c, s.store)
+func TestPlacementPolicyCache(t *testing.T) {
+	store, close := testkit.CreateMockStore(t)
+	defer close()
+	dom, err := session.BootstrapSession(store)
```

Comment:
Simple code like this.
```
	store, dom, clean := testkit.CreateMockStoreAndDomain(t)
	defer clean()
```

ditto other.

Reply:
Done